Create b2b-operations.md (#7808)
auth0/docs: articles/architecture-scenarios/implementation/b2b/b2b-operations.md

---
title: Operations
description: Understand operationalization of your Auth0 tenant environments
toc: true
topics:
    - b2b
    - b2biam
    - tenants
    - operations
contentType: concept
useCase:
  - tenant-operations
---

# Operations

<%= include('../../_includes/_operations/_introduction.md', { platform: 'b2b' }) %>

## Service status

<%= include('../../_includes/_operations/_service-status.md', { platform: 'b2b' }) %>

## Email provider setup

<%= include('../../_includes/_operations/_email-provider.md', { platform: 'b2b' }) %>

## Infrastructure

<%= include('../../_includes/_operations/_infrastructure.md', { platform: 'b2b' }) %>

## Logging

<%= include('../../_includes/_operations/_logging.md', { platform: 'b2b' }) %>

## Monitoring

<%= include('../../_includes/_operations/_monitoring.md', { platform: 'b2b' }) %>

## Notifications

<%= include('../../_includes/_operations/_notifications.md', { platform: 'b2b' }) %>

## Self-Service IDP provisioning

> Auth0 makes it easy to configure IDPs, but depending on the number of organizations and the amount of coordination work with those organizations, it can be a time consuming process to onboard several new customer organization IDPs regularly.  As a result, many of our customers have found it worth the time and effort to build a self-service portal for the administrators of their customers' organization so that they can configure their own IDP instead of having your IT department have to work with them.  Auth0’s Management API provides all of the necessary functionality to support your UI.

Reviewer: ```suggestion
Whilst Auth0 [Connections](identityproviders) makes it easy to configure Identity Providers (IdPs), it can be a time consuming process to onboard customer organization IdPs - especially if you are selling to new customer organizations on a regular basis, or existing customer organizations have changing IdP requirements. As a result, many of our customers have found it worth the time and effort to build a self-service portal for the administrators of their customers' organization, so that they can configure their own IdP - instead of having your IT department do all the work. The [Auth0 Management API](/api/management/v2) can be leveraged in support of this, and provides all of the necessary [Connection](/api/management/v2#!/Connections/get_connections) management functionality, et al, required to achieve this.
```

Author: Added to a separate PR

---
title: Operations
description: Understand operationalization of your Auth0 tenant environments
toc: true
topics:
    - b2b
    - b2biam
    - tenants
    - operations
contentType: concept
useCase:
  - tenant-operations
---

# Operations

<%= include('../../_includes/_operations/_introduction.md', { platform: 'b2b' }) %>

## Service status

<%= include('../../_includes/_operations/_service-status.md', { platform: 'b2b' }) %>

## Email provider setup

<%= include('../../_includes/_operations/_email-provider.md', { platform: 'b2b' }) %>

## Infrastructure

<%= include('../../_includes/_operations/_infrastructure.md', { platform: 'b2b' }) %>

## Logging

<%= include('../../_includes/_operations/_logging.md', { platform: 'b2b' }) %>

## Monitoring

<%= include('../../_includes/_operations/_monitoring.md', { platform: 'b2b' }) %>

## Notifications

<%= include('../../_includes/_operations/_notifications.md', { platform: 'b2b' }) %>

## Self-Service IDP provisioning

> Auth0 makes it easy to configure IDPs, but depending on the number of organizations and the amount of coordination work with those organizations, it can be a time consuming process to onboard a new customer organization IDP.  As a result, many of our customers have found it worth the time and effort to build a self-service portal for the administrators of their customer organizations so that they can configure their own IDP instead of having your IT department have to work with them.  Auth0’s Management API provides all of the necessary functionality to support your UI.

Reviewer 2: ```suggestion
Auth0 makes it easy to configure IDPs, but depending on the number of organizations and the amount of coordination work with those organizations, it can be a time consuming process to onboard several new customer organization IDPs regularly.  As a result, many of our customers have found it worth the time and effort to build a self-service portal for the administrators of their customers' organization so that they can configure their own IDP instead of having your IT department have to work with them.  Auth0’s Management API provides all of the necessary functionality to support your UI.
```

> ::: Best practice

Reviewer 2: ```suggestion
```

> If you are creating a separate Auth0 tenant for your more [complicated organizations](/architecture-scenarios/b2b/b2b-architecture#tenant-provision-for-complex-organizations), then you can consider allowing an administrator from that organization to have access to the Auth0 Dashboard of that separate tenant.  This would depend on how much trust there is with the other organization as that administrator would be able to break their login experience if they don’t know how to set things up correctly.

Reviewer 2: ```suggestion
```

> :::

Reviewer 2: ```suggestion
```

I think we've decided we don't want to recommend giving dashboard admin privileges to people outside of your organization.
